aries-site/src/examples/templates/forms/RequiredFieldsExample.js

```diff
@@ -0,0 +1,172 @@
+    setFormValues(value);
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Text size="xxlarge" weight="bold">
+            Form Header
+          </Text>
+        </Header>
+        <Box
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            messages={{
+              required: (
+                // need to define background otherwise
+                // inherits validation-critical background
+                <Error background="background-front">
+                  This is a required field.
+                </Error>
+              ),
+            }}
+            value={formValues}
+            onChange={onFormChange}
+          >
+            <RequiredFormField
+              required
+              error={
+                <Error background="background-front">
+                  Provide a unique name.
+                </Error>
+              }
+              htmlFor="name__input"
+              name="name"
+              label="Name"
+            >
+              <TextInput id="name" name="name" />
+            </RequiredFormField>
+            <RequiredFormField
+              required
+              htmlFor="superPower__input"
+              name="superPower"
+              label="Superpower"
+            >
+              <Select options={superPower} id="superPower" name="superPower" />
+            </RequiredFormField>
+            <RequiredFormField
+              required
+              htmlFor="weakness__input"
+              name="weakness"
+              label="Weakness"
+            >
+              <Select options={weakness} id="weakness" name="weakness" />
+            </RequiredFormField>
+            <RequiredFormField
+              required
+              htmlFor="email__input"
+              name="email"
+              label="Email"
+            >
+              <TextInput id="email" name="email" />
+            </RequiredFormField>
+            <RequiredFormField
+              required
+              help="Would you like to apply nemesis character"
+              htmlFor="nemesis__input"
+              name="nemesis"
+              label="Nemesis"
+            >
+              <CheckBox name="nemesis" label="Bring it on" toggle reverse />
+            </RequiredFormField>
+            <FormField htmlFor="comments" name="comments" label="Comments">
+              <TextArea id="comments" name="comments" placeholder="Comments" />
+            </FormField>
+            <Box
+              direction="row"
+              margin={{ top: 'medium', bottom: 'medium' }}
+              gap="xsmall"
+              round="4px"
+              pad="small"
+              background="validation-critical"
+            >
+              <Error>
```

Comment:
[Related](https://github.com/hpe-design/design-system/pull/1000/files#r465384018)

Supply background color here.
```suggestion
              <Error background="validation-critical" >
```